modules/cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -96,6 +96,49 @@ spec:
 $ oc create -f <file-name>.yaml
 ----
 
+[id=cluster-logging-collector-log-forward-examples-syslog-log-source]
+== Adding log source information to message output
+
+You can add `namespace_name`, `pod_name`, and `container_name` elements to the `message` field of the record by adding the `AddLogSource` field to your `ClusterLogForwarder` custom resource (CR).
+
+[source,yaml]
+----
+  spec:
+    outputs:
+    - name: syslogout
+      syslog:
+        addLogSource: true
+        facility: user
+        payloadKey: message
+        rfc: RFC3164
+        severity: debug
+        tag: mytag
+      type: syslog
+      url: tls://syslog-receiver.openshift-logging.svc:24224
+    pipelines:
+    - inputRefs:
+      - application
+      name: test-app
+      outputRefs:
+      - syslogout
+----
+
+[NOTE]
+====
+This configuration is compatible with both RFC3164 and RFC5424.
+====
+
+Syslog message output without `AddLogSource`:
+----
+<15>1 2020-11-15T17:06:14+00:00 fluentd-9hkb4 mytag - - -  {"msgcontent"=>"Message Contents", "timestamp"=>"2020-11-15 17:06:09", "tag_key"=>"rec_tag", "index"=>56}
+----
+.Example syslog message output with `AddLogSource`
+
+[source,json]
+----
+2022-03-02T15:52:11+00:00 logci2904-svsbv-worker-c-ssdrn.c.openshift-qe.internal mytag: {"docker":{"container_id":"fe7fb4e0a6e3140cbecdff91863eb17d9136db5afa1be4424af6d3407f835e91"},"kubernetes":{"container_name":"centos-logtest","namespace_name":"npzon","pod_name":"centos-logtest-dzjwp","container_image":"quay.io/openshifttest/ocp-logtest@sha256:16232868ba1143721b786dbabb3f7384645acb663fadb4af48e9ea1228a67635","container_image_id":"quay.io/openshifttest/ocp-logtest@sha256:16232868ba1143721b786dbabb3f7384645acb663fadb4af48e9ea1228a67635","pod_id":"3d106242-b12c-4b5f-83e3-99d81b03c305","pod_ip":"10.129.2.77","host":"logci2904-svsbv-worker-c-ssdrn.c.openshift-qe.internal","labels":{"run":"centos-logtest","test":"centos-logtest"},"master_url":"https://kubernetes.default.svc","namespace_id":"a2c56119-1f2b-420f-9b0b-9cff6531e179","namespace_labels":{"kubernetes_io/metadata_name":"npzon"}},"message":"namespace_name=npzon, container_name=centos-logtest, pod_name=centos-logtest-dzjwp, message={\"message\":\"MERGE_JSON_LOG=true\",\"level\":\"debug\",\"Layer1\":\"layer1 0\",\"layer2\":{\"name\":\"Layer2 1\",\"tips\":\"Decide by PRESERVE_JSON_LOG\"},\"StringNumber\":\"10\",\"Number\":10,\"foo.bar\":\"Dot Item\",\"{foobar}\":\"Brace Item\",\"[foobar]\":\"Bracket Item\",\"foo:bar\":\"Colon Item\",\"foo bar\":\"Space Item\"}","level":"unknown","hostname":"logci2904-svsbv-worker-c-ssdrn.c.openshift-qe.internal","pipeline_metadata":{"collector":{"ipaddr4":"10.0.128.3","inputname":"fluent-plugin-systemd","name":"fluentd","received_at":"2022-03-02T15:50:45.549186+00:00","version":"1.7.4 1.6.0"}},"@timestamp":"2022-03-02T15:50:45.547394+00:00","viaq_index_name":"app-write","viaq_msg_id":"Yjc0ZWZmNTYtNGI2MC00MjgwLTk3NTMtZjhlMTQ0MTQ2NTI4","log_type":"application"}
+----
```

Comment:
this does not seem correct. Please check description of this PR
https://github.com/openshift/cluster-logging-operator/pull/790